[2025-03-25T00:00:29.857+0000] {processor.py:186} INFO - Started process (PID=656) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:00:29.858+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:00:29.862+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:00:29.861+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:00:29.877+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:00:29.926+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:00:29.926+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:00:29.973+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:00:29.973+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:00:30.015+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.170 seconds
[2025-03-25T00:01:00.376+0000] {processor.py:186} INFO - Started process (PID=672) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:01:00.378+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:01:00.382+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:01:00.381+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:01:00.401+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:01:00.456+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:01:00.454+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:01:00.516+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:01:00.515+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:01:00.552+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.185 seconds
[2025-03-25T00:01:05.259+0000] {processor.py:186} INFO - Started process (PID=682) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:01:05.260+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:01:05.263+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:01:05.262+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:01:05.273+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:01:05.270+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 19, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['$(pwd)/../in:/usr/src/app/in', '$(pwd):/usr/src/app']}
[2025-03-25T00:01:05.274+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:01:05.307+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.056 seconds
[2025-03-25T00:01:35.670+0000] {processor.py:186} INFO - Started process (PID=698) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:01:35.674+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:01:35.677+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:01:35.677+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:01:35.685+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:01:35.683+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 19, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['$(pwd)/../in:/usr/src/app/in', '$(pwd):/usr/src/app']}
[2025-03-25T00:01:35.686+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:01:35.713+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.050 seconds
[2025-03-25T00:02:06.037+0000] {processor.py:186} INFO - Started process (PID=714) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:02:06.040+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:02:06.046+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:02:06.046+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:02:06.054+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:02:06.052+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 19, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['$(pwd)/../in:/usr/src/app/in', '$(pwd):/usr/src/app']}
[2025-03-25T00:02:06.055+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:02:06.081+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.053 seconds
[2025-03-25T00:02:36.374+0000] {processor.py:186} INFO - Started process (PID=730) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:02:36.375+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:02:36.378+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:02:36.377+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:02:36.391+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:02:36.387+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 19, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['$(pwd)/../in:/usr/src/app/in', '$(pwd):/usr/src/app']}
[2025-03-25T00:02:36.393+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:02:36.433+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.069 seconds
[2025-03-25T00:03:06.771+0000] {processor.py:186} INFO - Started process (PID=746) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:03:06.776+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:03:06.778+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:03:06.778+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:03:06.790+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:03:06.788+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 19, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['$(pwd)/../in:/usr/src/app/in', '$(pwd):/usr/src/app']}
[2025-03-25T00:03:06.791+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:03:06.822+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.060 seconds
[2025-03-25T00:03:37.156+0000] {processor.py:186} INFO - Started process (PID=762) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:03:37.158+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:03:37.159+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:03:37.159+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:03:37.168+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:03:37.166+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 19, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['$(pwd)/../in:/usr/src/app/in', '$(pwd):/usr/src/app']}
[2025-03-25T00:03:37.169+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:03:37.194+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.046 seconds
[2025-03-25T00:04:07.511+0000] {processor.py:186} INFO - Started process (PID=778) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:04:07.512+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:04:07.515+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:04:07.514+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:04:07.526+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:04:07.524+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 19, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['$(pwd)/../in:/usr/src/app/in', '$(pwd):/usr/src/app']}
[2025-03-25T00:04:07.528+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:04:07.564+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.061 seconds
[2025-03-25T00:04:19.629+0000] {processor.py:186} INFO - Started process (PID=783) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:04:19.629+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:04:19.631+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:04:19.631+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:04:19.641+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:04:19.638+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['/caminho/absoluto/para/in:/usr/src/app/in', '/caminho/absoluto/para/app:/usr/src/app']}
[2025-03-25T00:04:19.642+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:04:19.673+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.051 seconds
[2025-03-25T00:04:50.033+0000] {processor.py:186} INFO - Started process (PID=799) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:04:50.034+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:04:50.037+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:04:50.036+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:04:50.045+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:04:50.042+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['/caminho/absoluto/para/in:/usr/src/app/in', '/caminho/absoluto/para/app:/usr/src/app']}
[2025-03-25T00:04:50.046+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:04:50.069+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.043 seconds
[2025-03-25T00:05:20.410+0000] {processor.py:186} INFO - Started process (PID=816) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:05:20.414+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:05:20.416+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:05:20.416+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:05:20.426+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:05:20.423+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['/caminho/absoluto/para/in:/usr/src/app/in', '/caminho/absoluto/para/app:/usr/src/app']}
[2025-03-25T00:05:20.427+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:05:20.451+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.050 seconds
[2025-03-25T00:05:35.619+0000] {processor.py:186} INFO - Started process (PID=832) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:05:35.620+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:05:35.621+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:05:35.621+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:05:35.631+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:05:35.628+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['/home/andre/www/example_comunication_docker/example_apache_airflow/in:/usr/src/app/in', '/www/example_comunication_docker/example_containers/php-app2:/usr/src/app']}
[2025-03-25T00:05:35.632+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:05:35.656+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.046 seconds
[2025-03-25T00:05:54.823+0000] {processor.py:186} INFO - Started process (PID=843) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:05:54.827+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:05:54.829+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:05:54.828+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:05:54.843+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:05:54.840+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['/home/andre/www/example_comunication_docker/example_apache_airflow/in:/usr/src/app/in', '/www/example_comunication_docker/example_containers/php-app1:/usr/src/app']}
[2025-03-25T00:05:54.845+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:05:54.874+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.058 seconds
[2025-03-25T00:06:25.224+0000] {processor.py:186} INFO - Started process (PID=859) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:06:25.226+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:06:25.228+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:06:25.227+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:06:25.254+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:06:25.242+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['/home/andre/www/example_comunication_docker/example_apache_airflow/in:/usr/src/app/in', '/www/example_comunication_docker/example_containers/php-app1:/usr/src/app']}
[2025-03-25T00:06:25.258+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:06:25.308+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.097 seconds
[2025-03-25T00:06:31.358+0000] {processor.py:186} INFO - Started process (PID=864) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:06:31.361+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:06:31.363+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:06:31.363+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:06:31.372+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:06:31.370+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['www/example_comunication_docker/example_containersin:/usr/src/app/in', '/www/example_comunication_docker/example_containers/php-app1:/usr/src/app']}
[2025-03-25T00:06:31.373+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:06:31.399+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.050 seconds
[2025-03-25T00:06:52.627+0000] {processor.py:186} INFO - Started process (PID=874) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:06:52.628+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:06:52.630+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:06:52.629+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:06:52.639+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:06:52.637+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['/home/andre/www/example_comunication_docker/example_containers/in:/usr/src/app/in', '/home/andre/www/example_comunication_docker/example_containers/php-app1:/usr/src/app']}
[2025-03-25T00:06:52.640+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:06:52.664+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.045 seconds
[2025-03-25T00:12:02.579+0000] {processor.py:186} INFO - Started process (PID=170) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:12:02.581+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:12:02.585+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:12:02.584+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:12:02.600+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:12:02.596+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['/home/andre/www/example_comunication_docker/example_containers/in:/usr/src/app/in', '/home/andre/www/example_comunication_docker/example_containers/php-app1:/usr/src/app']}
[2025-03-25T00:12:02.603+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:12:02.647+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.079 seconds
[2025-03-25T00:12:32.831+0000] {processor.py:186} INFO - Started process (PID=186) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:12:32.833+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:12:32.836+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:12:32.836+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:12:32.857+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:12:32.851+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['/home/andre/www/example_comunication_docker/example_containers/in:/usr/src/app/in', '/home/andre/www/example_comunication_docker/example_containers/php-app1:/usr/src/app']}
[2025-03-25T00:12:32.859+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:12:32.910+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.091 seconds
[2025-03-25T00:13:03.246+0000] {processor.py:186} INFO - Started process (PID=202) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:13:03.250+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:13:03.253+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:13:03.252+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:13:03.265+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:13:03.263+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['/home/andre/www/example_comunication_docker/example_containers/in:/usr/src/app/in', '/home/andre/www/example_comunication_docker/example_containers/php-app1:/usr/src/app']}
[2025-03-25T00:13:03.266+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:13:03.301+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.065 seconds
[2025-03-25T00:13:33.687+0000] {processor.py:186} INFO - Started process (PID=218) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:13:33.690+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:13:33.694+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:13:33.693+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:13:33.702+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:13:33.699+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['/home/andre/www/example_comunication_docker/example_containers/in:/usr/src/app/in', '/home/andre/www/example_comunication_docker/example_containers/php-app1:/usr/src/app']}
[2025-03-25T00:13:33.703+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:13:33.727+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.051 seconds
[2025-03-25T00:14:04.047+0000] {processor.py:186} INFO - Started process (PID=234) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:14:04.051+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:14:04.054+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:14:04.054+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:14:04.064+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:14:04.061+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['/home/andre/www/example_comunication_docker/example_containers/in:/usr/src/app/in', '/home/andre/www/example_comunication_docker/example_containers/php-app1:/usr/src/app']}
[2025-03-25T00:14:04.065+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:14:04.093+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.054 seconds
[2025-03-25T00:14:34.401+0000] {processor.py:186} INFO - Started process (PID=250) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:14:34.406+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:14:34.412+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:14:34.411+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:14:34.429+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:14:34.425+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['/home/andre/www/example_comunication_docker/example_containers/in:/usr/src/app/in', '/home/andre/www/example_comunication_docker/example_containers/php-app1:/usr/src/app']}
[2025-03-25T00:14:34.432+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:14:34.477+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.088 seconds
[2025-03-25T00:14:54.726+0000] {processor.py:186} INFO - Started process (PID=266) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:14:54.730+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:14:54.733+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:14:54.733+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:14:54.742+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:14:54.740+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['/home/andre/www/example_comunication_docker/example_containers/in:/usr/src/app/in:rw', '/home/andre/www/example_comunication_docker/example_containers/php-app1:/usr/src/app:rw']}
[2025-03-25T00:14:54.743+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:14:54.796+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.078 seconds
[2025-03-25T00:15:25.207+0000] {processor.py:186} INFO - Started process (PID=282) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:15:25.213+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:15:25.219+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:15:25.218+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:15:25.253+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:15:25.241+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['/home/andre/www/example_comunication_docker/example_containers/in:/usr/src/app/in:rw', '/home/andre/www/example_comunication_docker/example_containers/php-app1:/usr/src/app:rw']}
[2025-03-25T00:15:25.256+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:15:25.329+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.135 seconds
[2025-03-25T00:15:55.672+0000] {processor.py:186} INFO - Started process (PID=298) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:15:55.677+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:15:55.680+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:15:55.680+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:15:55.694+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:15:55.690+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['/home/andre/www/example_comunication_docker/example_containers/in:/usr/src/app/in:rw', '/home/andre/www/example_comunication_docker/example_containers/php-app1:/usr/src/app:rw']}
[2025-03-25T00:15:55.696+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:15:55.733+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.071 seconds
[2025-03-25T00:16:26.052+0000] {processor.py:186} INFO - Started process (PID=314) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:16:26.055+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:16:26.059+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:16:26.059+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:16:26.070+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:16:26.068+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 24, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 256, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 959, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to DockerOperator (task_id: php_app1). Invalid arguments were:
**kwargs: {'volumes': ['/home/andre/www/example_comunication_docker/example_containers/in:/usr/src/app/in:rw', '/home/andre/www/example_comunication_docker/example_containers/php-app1:/usr/src/app:rw']}
[2025-03-25T00:16:26.072+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:16:26.098+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.057 seconds
[2025-03-25T00:16:35.216+0000] {processor.py:186} INFO - Started process (PID=319) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:16:35.220+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:16:35.223+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:16:35.223+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:16:35.238+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:16:35.234+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 36, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 263, in __init__
    raise ValueError(msg)
ValueError: Invalid `auto_remove` value True, expected one of 'never', 'success', or 'force'.
[2025-03-25T00:16:35.239+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:16:35.269+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.065 seconds
[2025-03-25T00:17:05.655+0000] {processor.py:186} INFO - Started process (PID=335) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:17:05.659+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:17:05.662+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:17:05.662+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:17:05.672+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:17:05.670+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 36, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 263, in __init__
    raise ValueError(msg)
ValueError: Invalid `auto_remove` value True, expected one of 'never', 'success', or 'force'.
[2025-03-25T00:17:05.673+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:17:05.699+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.053 seconds
[2025-03-25T00:17:36.053+0000] {processor.py:186} INFO - Started process (PID=351) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:17:36.054+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:17:36.057+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:17:36.056+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:17:36.065+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:17:36.063+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 36, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 263, in __init__
    raise ValueError(msg)
ValueError: Invalid `auto_remove` value True, expected one of 'never', 'success', or 'force'.
[2025-03-25T00:17:36.066+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:17:36.091+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.045 seconds
[2025-03-25T00:18:06.420+0000] {processor.py:186} INFO - Started process (PID=366) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:18:06.424+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:18:06.427+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:18:06.427+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:18:06.441+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:18:06.437+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/containers1_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/containers1_dag.py", line 36, in <module>
    task1 = DockerOperator(
            ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 508, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 263, in __init__
    raise ValueError(msg)
ValueError: Invalid `auto_remove` value True, expected one of 'never', 'success', or 'force'.
[2025-03-25T00:18:06.442+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:18:06.482+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.074 seconds
[2025-03-25T00:18:08.567+0000] {processor.py:186} INFO - Started process (PID=371) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:18:08.569+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:18:08.573+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:18:08.572+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:18:08.588+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:18:08.800+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:18:08.799+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG:containers1_dag
[2025-03-25T00:18:08.822+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:18:08.821+0000] {override.py:1912} INFO - Created Permission View: can edit on DAG:containers1_dag
[2025-03-25T00:18:08.837+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:18:08.836+0000] {override.py:1912} INFO - Created Permission View: can read on DAG:containers1_dag
[2025-03-25T00:18:08.856+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:18:08.855+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG Run:containers1_dag
[2025-03-25T00:18:08.867+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:18:08.867+0000] {override.py:1912} INFO - Created Permission View: menu access on DAG Run:containers1_dag
[2025-03-25T00:18:08.879+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:18:08.879+0000] {override.py:1912} INFO - Created Permission View: can create on DAG Run:containers1_dag
[2025-03-25T00:18:08.891+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:18:08.891+0000] {override.py:1912} INFO - Created Permission View: can read on DAG Run:containers1_dag
[2025-03-25T00:18:08.892+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:18:08.892+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:18:08.922+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:18:08.921+0000] {dag.py:3262} INFO - Creating ORM DAG for containers1_dag
[2025-03-25T00:18:08.936+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:18:08.936+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-01 00:00:00+00:00, run_after=2024-03-02 00:00:00+00:00
[2025-03-25T00:18:08.968+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.413 seconds
[2025-03-25T00:18:39.679+0000] {processor.py:186} INFO - Started process (PID=457) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:18:39.683+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:18:39.689+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:18:39.688+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:18:39.865+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:18:40.216+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:18:40.216+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:18:40.437+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:18:40.436+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:18:40.520+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.877 seconds
[2025-03-25T00:19:11.360+0000] {processor.py:186} INFO - Started process (PID=499) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:19:11.364+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:19:11.368+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:19:11.367+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:19:11.382+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:19:11.420+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:19:11.420+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:19:11.499+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.149 seconds
[2025-03-25T00:19:41.850+0000] {processor.py:186} INFO - Started process (PID=515) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:19:41.855+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:19:41.862+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:19:41.861+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:19:41.881+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:19:41.924+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:19:41.923+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:19:42.019+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.179 seconds
[2025-03-25T00:20:12.315+0000] {processor.py:186} INFO - Started process (PID=531) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:20:12.316+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:20:12.320+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:20:12.319+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:20:12.332+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:20:12.366+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:20:12.366+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:20:12.441+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.140 seconds
[2025-03-25T00:20:42.813+0000] {processor.py:186} INFO - Started process (PID=547) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:20:42.825+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:20:42.831+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:20:42.829+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:20:42.864+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:20:42.923+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:20:42.923+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:20:43.037+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.238 seconds
[2025-03-25T00:21:13.352+0000] {processor.py:186} INFO - Started process (PID=563) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:21:13.356+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:21:13.358+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:21:13.358+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:21:13.370+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:21:13.402+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:21:13.401+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:21:13.463+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.122 seconds
[2025-03-25T00:21:43.818+0000] {processor.py:186} INFO - Started process (PID=579) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:21:43.822+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:21:43.826+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:21:43.825+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:21:43.841+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:21:43.876+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:21:43.876+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:21:43.942+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.136 seconds
[2025-03-25T00:22:06.074+0000] {processor.py:186} INFO - Started process (PID=595) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:22:06.075+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:22:06.077+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:22:06.077+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:22:06.090+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:22:06.127+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:22:06.127+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:22:06.188+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.123 seconds
[2025-03-25T00:22:36.524+0000] {processor.py:186} INFO - Started process (PID=611) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:22:36.528+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:22:36.531+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:22:36.531+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:22:36.544+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:22:36.635+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:22:36.634+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:22:36.717+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.200 seconds
[2025-03-25T00:23:07.059+0000] {processor.py:186} INFO - Started process (PID=627) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:23:07.063+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:23:07.066+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:23:07.066+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:23:07.079+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:23:07.111+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:23:07.110+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:23:07.150+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:23:07.150+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-01 00:00:00+00:00, run_after=2024-03-02 00:00:00+00:00
[2025-03-25T00:23:07.181+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.130 seconds
[2025-03-25T00:24:45.428+0000] {processor.py:186} INFO - Started process (PID=171) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:24:45.430+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:24:45.434+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:24:45.433+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:24:45.528+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:24:46.221+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:24:46.220+0000] {override.py:1912} INFO - Created Permission View: can read on DAG:containers1_dag
[2025-03-25T00:24:46.246+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:24:46.245+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG:containers1_dag
[2025-03-25T00:24:46.265+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:24:46.264+0000] {override.py:1912} INFO - Created Permission View: can edit on DAG:containers1_dag
[2025-03-25T00:24:46.291+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:24:46.290+0000] {override.py:1912} INFO - Created Permission View: can read on DAG Run:containers1_dag
[2025-03-25T00:24:46.308+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:24:46.308+0000] {override.py:1912} INFO - Created Permission View: menu access on DAG Run:containers1_dag
[2025-03-25T00:24:46.325+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:24:46.325+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG Run:containers1_dag
[2025-03-25T00:24:46.343+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:24:46.342+0000] {override.py:1912} INFO - Created Permission View: can create on DAG Run:containers1_dag
[2025-03-25T00:24:46.343+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:24:46.343+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:24:46.374+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:24:46.374+0000] {dag.py:3262} INFO - Creating ORM DAG for containers1_dag
[2025-03-25T00:24:46.398+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:24:46.398+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-01 00:00:00+00:00, run_after=2024-03-02 00:00:00+00:00
[2025-03-25T00:24:46.435+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 1.018 seconds
[2025-03-25T00:25:16.710+0000] {processor.py:186} INFO - Started process (PID=187) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:25:16.714+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:25:16.718+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:25:16.717+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:25:16.735+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:25:16.780+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:25:16.780+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:25:16.817+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:25:16.817+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-01 00:00:00+00:00, run_after=2024-03-02 00:00:00+00:00
[2025-03-25T00:25:16.846+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.147 seconds
[2025-03-25T00:25:47.515+0000] {processor.py:186} INFO - Started process (PID=274) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:25:47.520+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:25:47.527+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:25:47.527+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:25:47.573+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:25:47.793+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:25:47.788+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:25:48.089+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:25:48.089+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:25:48.282+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.781 seconds
[2025-03-25T00:26:18.734+0000] {processor.py:186} INFO - Started process (PID=304) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:26:18.738+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:26:18.745+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:26:18.744+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:26:18.764+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:26:18.821+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:26:18.820+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:26:18.886+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:26:18.885+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:26:18.968+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.246 seconds
[2025-03-25T00:26:49.288+0000] {processor.py:186} INFO - Started process (PID=320) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:26:49.289+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:26:49.293+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:26:49.292+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:26:49.309+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:26:49.370+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:26:49.370+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:26:49.436+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:26:49.435+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:26:49.487+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.212 seconds
[2025-03-25T00:27:19.571+0000] {processor.py:186} INFO - Started process (PID=336) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:27:19.575+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:27:19.580+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:27:19.579+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:27:19.596+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:27:19.641+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:27:19.641+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:27:19.675+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:27:19.674+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:27:19.706+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.150 seconds
[2025-03-25T00:27:49.876+0000] {processor.py:186} INFO - Started process (PID=352) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:27:49.877+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:27:49.880+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:27:49.880+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:27:49.899+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:27:49.938+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:27:49.938+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:27:49.972+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:27:49.972+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:27:50.004+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.139 seconds
[2025-03-25T00:28:20.410+0000] {processor.py:186} INFO - Started process (PID=368) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:28:20.414+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:28:20.416+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:28:20.416+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:28:20.428+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:28:20.462+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:28:20.462+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:28:20.500+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:28:20.499+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:28:20.537+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.139 seconds
[2025-03-25T00:28:50.867+0000] {processor.py:186} INFO - Started process (PID=384) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:28:50.873+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:28:50.876+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:28:50.876+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:28:50.904+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:28:50.938+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:28:50.938+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:28:50.983+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:28:50.983+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:28:51.020+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.165 seconds
[2025-03-25T00:29:21.391+0000] {processor.py:186} INFO - Started process (PID=400) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:29:21.396+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:29:21.401+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:29:21.400+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:29:21.425+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:29:21.491+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:29:21.490+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:29:21.568+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:29:21.568+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:29:21.616+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.237 seconds
[2025-03-25T00:29:52.040+0000] {processor.py:186} INFO - Started process (PID=416) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:29:52.044+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:29:52.047+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:29:52.047+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:29:52.059+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:29:52.095+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:29:52.095+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:29:52.136+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:29:52.135+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:29:52.173+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.144 seconds
[2025-03-25T00:30:22.379+0000] {processor.py:186} INFO - Started process (PID=432) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:30:22.383+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:30:22.390+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:30:22.389+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:30:22.411+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:30:22.467+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:30:22.466+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:30:22.527+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:30:22.526+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:30:22.568+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.198 seconds
[2025-03-25T00:30:52.883+0000] {processor.py:186} INFO - Started process (PID=447) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:30:52.888+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:30:52.893+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:30:52.893+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:30:52.910+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:30:52.952+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:30:52.951+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:30:52.998+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:30:52.998+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:30:53.045+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.179 seconds
[2025-03-25T00:31:23.374+0000] {processor.py:186} INFO - Started process (PID=463) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:31:23.378+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:31:23.381+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:31:23.380+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:31:23.394+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:31:23.438+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:31:23.438+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:31:23.489+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:31:23.489+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:31:23.532+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.170 seconds
[2025-03-25T00:31:53.936+0000] {processor.py:186} INFO - Started process (PID=479) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:31:53.941+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:31:53.945+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:31:53.944+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:31:53.963+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:31:54.013+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:31:54.013+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:31:54.064+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:31:54.064+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:31:54.116+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.199 seconds
[2025-03-25T00:32:24.473+0000] {processor.py:186} INFO - Started process (PID=495) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:32:24.474+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:32:24.478+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:32:24.477+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:32:24.491+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:32:24.534+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:32:24.534+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:32:24.579+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:32:24.579+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:32:24.617+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.155 seconds
[2025-03-25T00:32:54.974+0000] {processor.py:186} INFO - Started process (PID=512) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:32:54.976+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T00:32:54.979+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:32:54.979+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:32:54.992+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T00:32:55.040+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:32:55.040+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T00:32:55.097+0000] {logging_mixin.py:190} INFO - [2025-03-25T00:32:55.097+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-15 00:00:00+00:00, run_after=2024-03-16 00:00:00+00:00
[2025-03-25T00:32:55.137+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.174 seconds
[2025-03-25T12:44:23.039+0000] {processor.py:186} INFO - Started process (PID=171) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:44:23.044+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T12:44:23.054+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:44:23.051+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:44:23.167+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:44:24.302+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:44:24.300+0000] {override.py:1912} INFO - Created Permission View: can edit on DAG:containers1_dag
[2025-03-25T12:44:24.330+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:44:24.330+0000] {override.py:1912} INFO - Created Permission View: can read on DAG:containers1_dag
[2025-03-25T12:44:24.383+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:44:24.382+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG:containers1_dag
[2025-03-25T12:44:24.440+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:44:24.440+0000] {override.py:1912} INFO - Created Permission View: can create on DAG Run:containers1_dag
[2025-03-25T12:44:24.472+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:44:24.470+0000] {override.py:1912} INFO - Created Permission View: can read on DAG Run:containers1_dag
[2025-03-25T12:44:24.502+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:44:24.500+0000] {override.py:1912} INFO - Created Permission View: menu access on DAG Run:containers1_dag
[2025-03-25T12:44:24.550+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:44:24.550+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG Run:containers1_dag
[2025-03-25T12:44:24.552+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:44:24.551+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T12:44:24.610+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:44:24.609+0000] {dag.py:3262} INFO - Creating ORM DAG for containers1_dag
[2025-03-25T12:44:24.664+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:44:24.663+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-01 00:00:00+00:00, run_after=2024-03-02 00:00:00+00:00
[2025-03-25T12:44:24.829+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 1.819 seconds
[2025-03-25T12:44:55.093+0000] {processor.py:186} INFO - Started process (PID=187) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:44:55.095+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T12:44:55.098+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:44:55.098+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:44:55.114+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:44:55.156+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:44:55.156+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T12:44:55.203+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:44:55.203+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-01 00:00:00+00:00, run_after=2024-03-02 00:00:00+00:00
[2025-03-25T12:44:55.246+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.162 seconds
[2025-03-25T12:45:25.556+0000] {processor.py:186} INFO - Started process (PID=203) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:45:25.557+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T12:45:25.560+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:45:25.560+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:45:25.573+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:45:25.608+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:45:25.607+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T12:45:25.649+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:45:25.649+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-01 00:00:00+00:00, run_after=2024-03-02 00:00:00+00:00
[2025-03-25T12:45:25.683+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.137 seconds
[2025-03-25T12:45:56.225+0000] {processor.py:186} INFO - Started process (PID=286) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:45:56.227+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T12:45:56.230+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:45:56.230+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:45:56.242+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:45:56.278+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:45:56.278+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T12:45:56.313+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:45:56.313+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-12 00:00:00+00:00, run_after=2024-03-13 00:00:00+00:00
[2025-03-25T12:45:56.340+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.128 seconds
[2025-03-25T12:46:26.664+0000] {processor.py:186} INFO - Started process (PID=302) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:46:26.665+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T12:46:26.668+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:46:26.667+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:46:26.682+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:46:26.729+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:46:26.728+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T12:46:26.762+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:46:26.762+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-12 00:00:00+00:00, run_after=2024-03-13 00:00:00+00:00
[2025-03-25T12:46:26.796+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.144 seconds
[2025-03-25T12:46:57.163+0000] {processor.py:186} INFO - Started process (PID=318) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:46:57.167+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T12:46:57.170+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:46:57.170+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:46:57.184+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:46:57.232+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:46:57.231+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T12:46:57.295+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:46:57.294+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-12 00:00:00+00:00, run_after=2024-03-13 00:00:00+00:00
[2025-03-25T12:46:57.326+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.176 seconds
[2025-03-25T12:47:27.667+0000] {processor.py:186} INFO - Started process (PID=333) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:47:27.669+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T12:47:27.672+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:47:27.672+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:47:27.694+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:47:27.752+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:47:27.752+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T12:47:27.815+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:47:27.815+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-12 00:00:00+00:00, run_after=2024-03-13 00:00:00+00:00
[2025-03-25T12:47:27.891+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.238 seconds
[2025-03-25T12:47:58.227+0000] {processor.py:186} INFO - Started process (PID=348) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:47:58.231+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T12:47:58.238+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:47:58.237+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:47:58.254+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:47:58.304+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:47:58.303+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T12:47:58.372+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:47:58.371+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-12 00:00:00+00:00, run_after=2024-03-13 00:00:00+00:00
[2025-03-25T12:47:58.415+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.202 seconds
[2025-03-25T12:48:28.766+0000] {processor.py:186} INFO - Started process (PID=364) to work on /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:48:28.768+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/containers1_dag.py for tasks to queue
[2025-03-25T12:48:28.774+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:48:28.774+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:48:28.795+0000] {processor.py:925} INFO - DAG(s) 'containers1_dag' retrieved from /opt/airflow/dags/containers1_dag.py
[2025-03-25T12:48:28.867+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:48:28.867+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-25T12:48:28.964+0000] {logging_mixin.py:190} INFO - [2025-03-25T12:48:28.963+0000] {dag.py:4180} INFO - Setting next_dagrun for containers1_dag to 2024-03-12 00:00:00+00:00, run_after=2024-03-13 00:00:00+00:00
[2025-03-25T12:48:29.018+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/containers1_dag.py took 0.263 seconds
